akka-actor-typed-tests/src/test/scala/akka/actor/typed/AskSpec.scala

```diff
@@ -51,12 +51,12 @@ class AskSpec extends ScalaTestWithActorTestKit("""
   "Ask pattern" must {
     "fail the future if the actor is already terminated" in {
       val ref = spawn(behavior)
-      (ref ? Stop).futureValue
+      (ref.ask(Stop)).futureValue
```

Comment:
in some of Arnout's 2.13 PRs I have seen that he added `.apply`, not needed here?

otherwise `Stop(_)` isn't too bad

Reply:
One param case class message works fine like this.

Reply:
I thought it was, but I was wrong, it gets inferred to `Nothing` here. :/